woocommerce/google-listings-and-ads
# Use woocommerce_gla as filter and action prefix

### Changes proposed in this Pull Request:

This PR standardizes the filter and action prefix. Since most of our hooks were already using the `woocommerce_gla` prefix, I decided to use the same here (this seems most inline with what is used in WC core).
Most of the hooks and filters are used internally so far, so it's best to change these as soon as possible.

Closes #741

### Detailed test instructions:

1. Add the right prefix to any custom code snippets like `add_filter( 'woocommerce_gla_enable_connection_test', '__return_true' );`
2. Test syncing products and editing GLA product attributes
3. Tryout some of the compatible plugins like Bundles and Brands
4. Confirm that setting of the attributes and syncing bundled products continues to work as intended
5. Check the WooCommerce > Status > Logs to confirm we are still getting new debug / error log entries

### Changelog entry
* Tweak - Standardize action and filter hook prefix.
